fe/fe-core/src/main/java/com/starrocks/external/hive/HiveRepository.java

```diff
@@ -38,7 +38,7 @@ public class HiveRepository {
     Map<String, HiveMetaCache> metaCaches = Maps.newHashMap();
     ReadWriteLock metaCachesLock = new ReentrantReadWriteLock();
 
-    Executor executor = Executors.newFixedThreadPool(100);
+    Executor executor = Executors.newCachedThreadPool();
```

Comment:
```suggestion
    Executor executor = return new ThreadPoolExecutor(<corePoolSize>, Integer.MAX_VALUE, 60L, TimeUnit.SECONDS, new SynchronousQueue<Runnable>());
```
<corePoolSize> need greater than 0. like 50 or 100

Reply:
executor threads is sed by `Config.hive_meta_cache_load_threads`, default value is 50